test/jdk/java/lang/reflect/records/R10.jcod

```diff
@@ -0,0 +1,259 @@
+class R10 {
```

Comment:
Please add some comments in the jcod file saying what it does/why it is being used.

Reply:
done